### PR TITLE
Redesign of the cycle switcher

### PR DESCRIPTION
**Pull request by tijmenb**
## Context

The current design of the functionality to switch cycles is a bonanza of buttons. @paulrobertlloyd did a nice redesign.

## Changes proposed in this pull request

image

## Guidance to review

OK?

## Link to Trello card

https://trello.com/c/hfddfSDo

## Things to check

- [X] This code does not rely on migrations in the same Pull Request
- [X] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [X] API release notes have been updated if necessary
- [X] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: config/locales/cycles.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      description: Candidates can no longer apply to unavailable courses
+    today_is_after_apply_2_deadline_passed:
+      name: Apply 2 deadline has passed
+      description: Candidates can no longer submit any subsequent applications
+    today_is_after_find_closes:
+      name: Find has closed
+      description: Candidates can no longer browse courses on Find
+    today_is_after_find_reopens:
+      name: Find has reopened
+      description: Candidates can browse course options on Find
```
Comment (paulrobertlloyd) [inline config/locales/cycles.yml:23]: Sorry, my bad:

```suggestion
      description: Candidates can browse courses on Find
```

**Review by paulrobertlloyd — APPROVED** [review]
Is it not possible to show what point in the cycle we are currently in within the hint text for ‘Same as production environment’?

File: config/locales/cycles.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+en:
+  cycles:
+    real:
+      name: 'Same as production environment'
+      description: ''
+    today_is_mid_cycle:
+      name: Mid cycle
+      description: Candidates can see upcoming deadlines
```
Comment (paulrobertlloyd) [inline config/locales/cycles.yml:8]:
```suggestion
      description: Candidates can see upcoming application deadlines
```

**Comment by tijmenb** [comment]
> Is it not possible to show what point in the cycle we are currently in within the hint text for ‘Same as production environment’?

No, that's a bit tricky atm. Perhaps later.

**Pull request merged**